### PR TITLE
Add BE-4 Metholox Engine from DylanSemrau

### PR DESCRIPTION
**Pull request by vader111**
Add BE-4 Metholox Engine from DylanSemrau.

### CODE REVIEW

**Comment by vader111** [comment]
Needs https://github.com/KSP-RO/RealismOverhaul/pull/2377

**Comment by vader111** [comment]
(Ready To Review)

**Comment by Capkirk123** [comment]
Looks good

File: GameData/ROEngines/PartConfigs/BE4_DylanSemrau.cfg [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	subcategory = 0
+	RSSROConfig = true
+	RP0conf = true
+
+	crashTolerance = 10
+	maxTemp = 673.15
+	skinTemp = 773.15
+	fuelCrossfeed = true
+	breakingForce = 10000
+	breakingTorque = 10000
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/BE4_DylanSemrau.cfg:17]: Make use of ROESetEngineDefaults

File: GameData/ROEngines/PartConfigs/BE4_DylanSemrau.cfg [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+	tags = BE-4, DylanSemrau, Blue Origin, Methalox
+
+	engineType = BE-4
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/BE4_DylanSemrau.cfg:49]: Make use of ROESetEngineDefaults

**Review by DRVeyl — CHANGES_REQUESTED** [review]

**Review by DRVeyl — APPROVED** [review]

**Pull request merged**